sesame/src/main/scala/org/w3/banana/sesame/extensions.scala

```diff
@@ -0,0 +1,42 @@
+package org.w3.banana
+package sesame
+
+import org.openrdf.model.Statement
+import org.openrdf.query.{GraphQueryResult, BindingSet, TupleQueryResult}
+import org.openrdf.repository.RepositoryResult
+
+/** small things to make life with Sesame easier */
+object extensions extends SesameResultsConversions
+
+
+trait SesameResultsConversions {
+
+  /**
+   * Implicit class that turns query result into iterator (so methods toList, map and so on can be applied to it)
+   *
+   * @param results results in sesame format
+   */
+  implicit class TupleResult(results: TupleQueryResult) extends Iterator[BindingSet]
```

Comment:
Just noticed that SolutionsSyntax is not enough =(( In many cases (at least with sesame) we have to manipulate con.getStatements() results that return repository result instead of solution